### PR TITLE
HP-1478: list the data fields service will use

### PR DESCRIPTION
**Pull request by NikoHelle**
List allowed data fields the service will use after user grants permissions.

Also fixed an accessibility problem: input field labels should not contain links. Moved links outside.

### CODE REVIEW

File: helsinki/login/messages/messages_sv.properties [hunk 1]
```diff
@@ -75,7 +76,7 @@ doGiveFeedback = Ge respons
 doGiveFeedbackLink = https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback
 profile = Profil
 profileHeader = En Helsingfors-profil håller på att skapas åt dig
-profileTermsText = SV: När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinki-profil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>Palvelut, joille annat luvan pääsevät lukemaan profiilin tietojasi.
+profileTermsText = När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinki-profil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>De tjänster du ger tillstånd till kan läsa information från din profil.
```
Comment (akikoskinen) [inline helsinki/login/messages/messages_sv.properties:79]: ...skapas det automatiskt en **Helsingforsprofil**...

**Review by akikoskinen — CHANGES_REQUESTED** [review]
One small translation bug, otherwise seems good.

File: helsinki/login/messages/messages_sv.properties (hunk 1 above)
Comment (NikoHelle) [reply helsinki/login/messages/messages_sv.properties]: Updated all sv translations where that text is used.

File: helsinki/login/messages/messages_sv.properties [hunk 2]
```diff
@@ -74,8 +75,8 @@ invalidEmailMessage = Ogiltigt e-post
 doGiveFeedback = Ge respons
 doGiveFeedbackLink = https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback
 profile = Profil
-profileHeader = En Helsingfors-profil håller på att skapas åt dig
-profileTermsText = SV: När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinki-profil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>Palvelut, joille annat luvan pääsevät lukemaan profiilin tietojasi.
+profileHeader = En Helsingforsprofil håller på att skapas åt dig
+profileTermsText = När du loggar in i Helsingfors tjänster skapas det automatiskt en Helsinforsprofil åt dig. Dina uppgifter finns behändigt tillgängliga på ett ställe. <br/><br/>De tjänster du ger tillstånd till kan läsa information från din profil.
```
Comment (akikoskinen) [inline helsinki/login/messages/messages_sv.properties:79]: Typo: Helsinforsprofil. 'g' missing.

**Review by akikoskinen — CHANGES_REQUESTED** [review]

File: helsinki/login/messages/messages_sv.properties (hunk 2 above)
Comment (NikoHelle) [reply helsinki/login/messages/messages_sv.properties]: 🤦‍♂️  Fixed.

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**